packages/core/src/db/entity/sqliteEntityStore.ts

```diff
@@ -218,9 +227,9 @@ export class SqliteEntityStore implements EntityStore {
         );
       }
 
-      const statement = `DELETE FROM "${entityName}" WHERE "id" = @id`;
+      const statement = `DELETE FROM "${entityName}" WHERE "id" = ?`;
 
-      const { changes } = this.db.prepare(statement).run({ id: id });
+      const { changes } = this.db.prepare(statement).run(id);
```

Comment:
The usages of `id` weren't strictly necessary, but I moved em over to `?` too for consistency.